src/components/Dashboard/Menu/OrganizationSwitcher.tsx

```diff
@@ -122,33 +141,14 @@ export const OrganizationSwitcher = ({ ...props }) => {
                 >
                   <Icon as={LuGalleryVerticalEnd} boxSize={4} ml={2} mr={2} />
                 </Flex>
-                {org.label}
-                {org.value === selectedOrg && (
-                  <Tag colorScheme='gray' ml='auto !important'>
-                    {t('current', { defaultValue: 'Current' })}
-                  </Tag>
-                )}
-              </Button>
-            ))
-          ) : (
-            <HStack p={2} mb={1}>
-              <Flex
-                justifyContent={'center'}
-                alignItems={'center'}
-                border='var(--border)'
-                w='22px'
-                h='22px'
-                borderRadius='xs'
-              >
-                <Icon as={LuGalleryVerticalEnd} boxSize={4} ml={2} mr={2} />
-              </Flex>
-              <OrganizationName fontSize={'14px'} lineHeight={'14px'} fontWeight={500} maxW={'80px'} isTruncated />
-              <Tag colorScheme='gray' ml='auto'>
-                {t('current', { defaultValue: 'Current' })}
-              </Tag>
-            </HStack>
-          )}
-        </Flex>
+                <OrganizationName fontSize={'14px'} lineHeight={'14px'} fontWeight={500} maxW={'80px'} isTruncated />
+                <Tag colorScheme='gray' ml='auto'>
+                  {t('current', { defaultValue: 'Current' })}
+                </Tag>
```

Comment:
Why is the current organization actually duplicated? Why not map through all the organizations when length is > 0 and avoid duplicating code?